application/actions/upload_test.go

```diff
@@ -0,0 +1,129 @@
+package actions
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"testing"
+	"time"
+
+	"github.com/gobuffalo/buffalo/binding"
+
+	"github.com/silinternational/wecarry-api/aws"
+	"github.com/silinternational/wecarry-api/domain"
+
+	"github.com/gobuffalo/httptest"
+	"github.com/silinternational/wecarry-api/models"
+)
+
+// UploadFixtures is for returning fixtures from Fixtures_Upload
+type UploadFixtures struct {
+	ClientID    string
+	AccessToken string
+}
+
+// Fixtures_Upload creates fixtures for the Test_Upload test
```

Comment:
Do you prefer having the fixture in the same file like this?  It seems neater to me to have it in a different file, so instead of scrolling up and down, you're just switching tabs.  Also, it means the test file stays cleaner.

Your choice.

Reply:
Yes, I do, for organizational clarity. I do appreciate your concern. Would an editor split view work for this?